packages/cli/internal/pkg/cli/clierror/cli_error_test.go

```diff
@@ -0,0 +1,35 @@
+package clierror
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/actionable"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_New_ActionableError(t *testing.T) {
```

Comment:
Could you please add a test case when Actionable Error wrapped in a generic error?

Reply:
There is already a test for this: Test_New_WithMatchingSuggestion

```
func Test_New_WithMatchingSuggestion(t *testing.T) {
	var errorMessageToSuggestedActionMap = map[string]string{
		"error occurred": "Please do this suggestion",
		"some error":     "Please do this other suggestion",
	}

	coreError := errors.New("my error occurred")
	actualError := FindSuggestionForError(coreError, errorMessageToSuggestedActionMap)

	expectedError := &Error{
		coreError,
		"Please do this suggestion",
	}

	assert.Equal(t, expectedError, actualError)
}
```